app/src/main/res/values/strings.xml

```diff
@@ -632,6 +639,12 @@
     <string name="settings_theme_heading">Appearance</string>
     <string name="settings_theme_desc">Dark/light theme</string>
 
+    <string name="settings_notification_heading">Choose Notification Action</string>
+    <string name="settings_notification_desc">Choose which action buttons to show with the RethinkDNS notification: %1$s</string>
+    <string name="settings_notification_desc1">Stop (default): Shows one button, tapping on which stops RethinkDNS</string>
+    <string name="settings_notification_desc2">DNS / DNS + Firewall: Shows two buttons to quickly switch between the battery-saver DNS mode and the regular DNS + Firewall mode.</string>
+    <string name="settings_notification_desc3">Empty: Shows no button.</string>
+
```

Comment:
where do these string appear? Can you post a screenshot, pls

Reply:
![Screenshot_20210522-022045](https://user-images.githubusercontent.com/56958445/119197420-4f6f5a80-baa5-11eb-9b6f-4925addccf85.png)

Reply:
Can we make something custom dialog?

Reply:
The description should appear how it does in the "Choose mode" bottom sheet on the homescreen. Also, rename "Empty" to "None".

```
None 
<desc-in-the-next-line>

DNS + Firewall
<desc-in-the-next-line>

Stop (default)
<desc-in-the-next-line>
```

Reply:
Added new-line. The Choose mode bottom sheet is custom layout. Check this  if its not good then we will create a custom layout for the dialog.

Reply:
![device-2021-05-23-144324](https://user-images.githubusercontent.com/56958445/119254680-4da4b480-bbd5-11eb-9019-62333f4f468d.png)

Reply:
Let me know if it's okay or we can go for the custom layout.

Reply:
Changed the notification title. 
The bottom sheet dialog layout are custom. Maybe we can try adding up some custom in later part. Now its a singleChoiceItem with default layout.